services/staging/rs_server_staging/main.py

```diff
@@ -72,95 +120,44 @@ def init_db():
-    except SQLAlchemyError as e:
-        logger.error(f"Error creating table or ENUM type in PostgreSQL: {e}")
-        raise RuntimeError(f"Error creating table or ENUM type in PostgreSQL: {e}") from e
-
-
-def get_manager_def():
-    """Loads and configures the PostgreSQL Manager definition for pygeoapi.
-
-    This function retrieves the manager definition from the pygeoapi configuration,
-    validates its structure, and dynamically replaces placeholder values in the
-    `connection` dictionary with environment variable values.
-
-    Behavior:
-        1. **Retrieve Manager Configuration**:
-           - Reads the `manager` configuration from the `api.config` object.
-
-        2. **Validate Configuration**:
-           - Ensures the manager definition and its `connection` key are properly structured
-             as dictionaries.
-
-        3. **Replace Placeholders with Environment Variables**:
-           - Iterates through the `connection` dictionary to replace any placeholders
-             (formatted as `${ENV_VAR}`) with corresponding values from environment variables.
-
-    Raises:
-        RuntimeError: If the manager definition is invalid, or if any required environment variable
-                      for a placeholder is missing.
+    Args:
+        pause: pause in seconds to wait for the database connection.
+        timeout: timeout in seconds to wait for the database connection.
 
     Returns:
-        dict: The validated and updated manager definition with all placeholders resolved.
-
-    Dependencies:
-        - Requires access to `api.config` for retrieving the initial configuration.
-        - Reads environment variables dynamically using `os.environ`.
-
-    Notes:
-        - The placeholders in the `connection` dictionary must follow the format `${ENV_VAR}`.
+        PostgreSQLManager instance
     """
-    manager_def = api.config.get("manager", {})
+    manager_def = api.config["manager"]
     if not manager_def or not isinstance(manager_def, dict) or not isinstance(manager_def["connection"], dict):
-        logger.error("Error reading the manager definition for pygeoapi PostgreSQL Manager")
-        raise RuntimeError("Error reading the manager definition for pygeoapi PostgreSQL Manager")
-    try:
-        for k, v in manager_def["connection"].items():
-            if v.startswith("${") and v.endswith("}"):
-                manager_def["connection"][k] = os.environ[f"{v[2:-1]}"]
-    except KeyError as e:
-        logger.error(f"Error when trying to read the environment variable: {e}")
-        raise RuntimeError(f"Error when trying to read the environment variable: {e}") from e
-    return manager_def
-
+        message = "Error reading the manager definition for pygeoapi PostgreSQL Manager"
+        logger.error(message)
+        raise RuntimeError(message)
+    connection = manager_def["connection"]
+
+    # Create SQL Alchemy engine
+    engine = get_engine(**connection)
```

Comment:
I am ok with getting the engine from the database.py. But again, moving the logic of the database structure outside the pythin code will easiness the maintaining of it as well as this part of the src which will be deleted